composer.json

```diff
@@ -25,9 +25,11 @@
         "ext-gmp": "*",
         "ext-intl": "*",
         "ext-json": "*",
+        "ext-libxml": "*",
```

Comment:
Although this is fine, I would have preferred another PR, as it is not related to the topic of the current PR.
No need to revert, though, but for another time.